src/database/DataTypes/TransactionItem.js

```diff
@@ -299,6 +299,16 @@ export class TransactionItem extends Realm.Object {
     return this.batches.sum('doses');
   }
 
+  get isInBreach() {
+    return this.batches.some(({ isInBreach }) => isInBreach);
+  }
```

Comment:
a `TransactionItem` being essentially a grouping of multiple batches can be in a breach if there is just one of the batches in a breach